### DISCUSSION TITLE
Logging

### DISCUSSION THREAD

**Discussion by jassherman** [ltb-project/self-service-password#987] [Q&A]
I'm on 1.6.1 in an ubi 8 container and get the extended error message displayed on the page but its not logging anywhere. On my server running 1.5.1 on al2, the error displays on the page and logs to the error log? any ideas or things to check? Ive got a multi-tenant setup and the setting below in each config.inc.<name>.php file. (i left the rest of the config files out)

$show_extended_error = true;
$audit_log_file = "/var/log/ssp/audit.log";
$reset_request_log = "/var/log/ssp";

**Comment by jassherman** [comment]
i found the logs i was looking for in /var/log/php-fpm/www-error.log.

**Reply by jassherman** [reply]
Hi Everyone,
I'm moving LTB Password from an AMI build (v1.5) on EC2 to an UBI container running php 8.3 and v1.6.1 and I'm trying to get the logs I need to STDOUT and STDERR. I've tried
creating a link between the php logs i want and /proc/self/fd/1 and 2 in the Dockerfile and that didnt work. i've tried this in the www.conf file php_admin_value[error_log] = /proc/self/fd/2, and this didnt work. 

Anyone have any luck with this?
Thanks in advance.

**Reply by coudot** [reply]
You can look at what we do in FusionIAM: https://gitlab.ow2.org/fusioniam/fusioniam/-/blob/master/build/rockylinux9/service-desk/ansible/run-ct.sh

**Reply by jassherman** [reply]
Thanks for the suggestion. When i link the logs to /proc/self/fd/1 or 2 (or /dev/stdout or stderr) the site has a 500 error. Not sure what is going on.
